client/package.json

```diff
@@ -298,6 +298,11 @@
         "title": "BitBake: Run task for recipe",
         "description": "Run a specific task for a bitbake recipe."
       },
+      {
+        "command": "bitbake.scan-recipe-env",
+        "title": "BitBake: Scan Recipe",
+        "description": "Scan the recipe for the complete information about the environment."
```

Comment:
is the hover definition the only thing we want from the scan results? I thought we might also make use of the include list or something.

Reply:
You're right. I still think we need a message that's more precise.
"Scan a recipe's environment: Variables, Include history, Classes" ?

Reply:
since it is running `bitbake -e`, we could just mention that and add the original or rephrased description from the `bitbake --help` which is: 
> Show the global or per-recipe environment complete with information about where variables were set/changed.

So in the end I will put something like this:
`"description": "Run 'bitbake -e' to get the environment about the recipe with complete information about where variables were set/change to enrich the language service such as hover definition"`

Reply:
Description was rephrased and updated.